### PR TITLE
Add initial portuguese translation to runtime class

### PR DESCRIPTION
**Pull request by rikatz**
Add initial translation to runtime class + necessary glossary

### CODE REVIEW

File: content/pt/docs/concepts/containers/runtime-class.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+suportem eles. Se o `scheduling` não estiver configurado, assume-se que esse RuntimeClass
+é suportado por todos os nós.
+
+Para garantir que os Pods sejam executados em um nó que suporte um RuntimeClass específico, 
+aquele conjunto de nós deve possuir uma marca/label padrão que é selecionado pelo campo 
+`runtimeclass.scheduling.nodeSelector`. O nodeSelector do RuntimeClass é combinado com o 
+nodeSelector do Pod em tempo de admissão, obtendo a intersecção do conjunto de nós selecionado 
+por cada. Se existir um conflito, o pod será rejeitado.
+
+Se os nós suportados possuírem marcação de restrição para previnir outros Pods com uma
```
Comment (jailton) [inline content/pt/docs/concepts/containers/runtime-class.md:149]: Just a little typo: previnir to prevenir.

File: content/pt/docs/concepts/containers/runtime-class.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+{{< note >}}
+RuntimeClass assume uma configuração homogênea de nós entre todo o cluster por padrão 
+(o que significa que todos os nós estão configurados do mesmo jeito referente aos agentes de 
+execução). Para suportar configurações heterogêneas, veja [Associação](#associação) abaixo.
+{{< /note >}}
+
+As configurações possuem um nome `handler` correspondente, referenciado pelo RuntimeClass. 
+Esse nome deve ser um valor DNS 1123 válido (letras, números e o carácter `-`).
+
+### 2. Crie o recurso RuntimeClass correspondente.
```
Comment (devlware) [inline content/pt/docs/concepts/containers/runtime-class.md:51]:
```suggestion
### 2. Crie o recurso RuntimeClass correspondente
```

File: content/pt/docs/concepts/containers/runtime-class.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+Você pode ainda usar um _RuntimeClass_ para executar diferentes Pods com o mesmo agente
+de execução de contêineres mas com diferentes configurações.
+
+## Configuração 
+
+1. Configure a implementação do CRI nos nós (depende do agente de execução)
+2. Crie o recurso RuntimeClass correspondente.
+
+### 1. Configure a implementação do CRI nos nós.
```
Comment (devlware) [inline content/pt/docs/concepts/containers/runtime-class.md:36]:
```suggestion
### 1. Configure a implementação do CRI nos nós
```

**Review by devlware — CHANGES_REQUESTED** [review]
Thanks @rikatz 🚀. I left some observations (nitpicks).

File: content/pt/docs/concepts/containers/runtime-class.md (hunk 1 above)
Comment (rikatz) [reply content/pt/docs/concepts/containers/runtime-class.md:149]:
```suggestion
Se os nós suportados possuírem marcação de restrição para prevenir outros Pods com uma 
```

**Comment by rikatz** [comment]
/label tide/merge-method-squash

@jailton @devlware thanks for the suggestion. I've commited them, and added the label to squash all the commits until the PR is accepted :)

**Pull request merged**